contact.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+	<link rel="stylesheet" href="css/all.min.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/contact.css">
+    <title>Document</title>
+</head>
+<body>
+   <!--HEADER section -->
+    <div class="container">
+        <header class="row">
+            <div class="col-xs-8 col-sm 4 col-md-6 centered">
+                <img src="img/petstore_logo.png" alt="Pet store logo" class="pull-left logo">
+                <h1 class="pull-left text-center">Pet Store</h1>
+            </div>
+            <nav class="col-xs-12 col-sm-7 col-md-6">
+                <ul class="nav nav-pills pull-right">
+                    <li role="presentation" >
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li role="presentation" class="active">
+                        <a href="#">Products</a>
+                    </li>
+                    <li role="presentation">
+                        <a href="contact.html">Contact Us</a>
+                    </li>
+                </ul>
+            </nav>
+        </header> 
+    </div>
+  
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12 col-md-6">
+                <div class="row">
```

Comment:
this row needs to have the divs with the img tag, the doctor's name, and p tag filled with lorem to live inside. Example: 
```suggestion
                <div class="row">
                     <div class="col-xs 6 col-sm-4">
                       <img src="img/doctor1.jpg" alt= "Dr. Smith" class="-responsive">
                    </div>
                    <div class="col-sx-6 col-sm-8">
                      <h3>Dr. Joe Smith</h3>
                    </div>
                    <div class="col-xs-12 col-sm-8">
                      <p class="lorem">Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis, nesciunt autem. Facilis minima ad libero, cum magni architecto itaque numquam facere, repudiandae quas totam sint dolore. Illum eum totam earum.</p>
                    </div>
               </div> //closes the row.     
```

Reply:
Okay thanks.  I just fixed it.  I appreciate your help!